src/index.ts

```diff
@@ -1,18 +1,7 @@
 "use strict";
```

Comment:
I used this for testing - it is recommended to have this when migration. I seen that some of those were removed. I will remove it everywhere later.